src/components/EventsList/index.tsx

```diff
@@ -1,14 +1,30 @@
 import styled from "@emotion/styled";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+interface categoryProps {
+  curCategory: number;
+}
 
 export default function EventsList() {
+  const [curCategory, setCurCategory] = useState(1);
+  const categories = ["all", "promotion", "collaboration"];
```

Comment:
T2: 타입 스크립트 쓰는 이점인 타입을 명시해주면 좋을 거 같습니다.
```suggestion
  const categories:string[] = ["all", "promotion", "collaboration"];
```

Reply:
이 부분은 type에러가 없어서 조금 찾아봤는데 TypeScript가 타입을 자동으로 추론하고 있는 경우에 해당되는 것 같아요! 아래 글 한번 참고해주세요 😊

[참고링크](https://velog.io/@teo/typescript#-%ED%95%98%EC%A7%80%EB%A7%8C-%ED%83%80%EC%9E%85-%EC%84%A0%EC%96%B8%EB%B3%B4%EB%8B%A4%EB%8A%94-%EA%B0%80%EA%B8%89%EC%A0%81-%EC%9E%90%EB%8F%99%EC%B6%94%EB%A1%A0%EC%9D%84-%EC%82%AC%EC%9A%A9%ED%95%98%EC%9E%90)